interfaces/matlab/toolbox/1D/CounterFlowDiffusionFlame.m

```diff
@@ -0,0 +1,203 @@
+function flame = CounterFlowDiffusionFlame(left, flow, right, tp_f, tp_o, oxidizer)
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% COUNTERFLOWDIFFUSIONFLAME  Create a counter flow diffusion flame stack.
+% flame = CounterFlowDiffusionFlame(left, flow, right, tp_f, tp_o, oxidizer)
+% :param left:
```

Comment:
To get this function in the Sphinx docs, it needs to be added in `doc/SConscript` (see the data structure near line 200)